cmd/livepeer/livepeer.go

```diff
@@ -287,6 +287,9 @@ func main() {
 	}
 
 	watcherErr := make(chan error)
+	var roundsWatcher *watchers.RoundsWatcher
+	var originalLastSeenBlock *big.Int
```

Comment:
Besides `roundsWatcher`, doesn't look like we need to hoist any of these variables up here anymore.

Reply:
fixed in d9c1a2d9